crates/common_utils/Cargo.toml

```diff
@@ -60,7 +60,7 @@ uuid = { version = "1.8.0", features = ["v7"] }
 # First party crates
 common_enums = { version = "0.1.0", path = "../common_enums" }
 masking = { version = "0.1.0", path = "../masking" }
-router_env = { version = "0.1.0", path = "../router_env", features = ["log_extra_implicit_fields", "log_custom_entries_to_extra"], optional = true, default-features = false }
+router_env = { version = "0.1.0", path = "../router_env", features = ["actix_web", "log_extra_implicit_fields", "log_custom_entries_to_extra"], default-features = false }
```

Comment:
We cannot be making `router_env` a required dependency here, since it would break WASM builds.